### PR TITLE
make port optional to setup_url_for_address

### PR DESCRIPTION
**Pull request by Spectre5**
## Description:
Make the `port` argument to `setup_url_for_address` optional by assigning a default value of `None`.  I'd think the vast majority of use cases would provide `None` for the argument anyway so that `pywemo` can probe for the port.

## Checklist:
  - [x] The code change is tested and works locally.
  - [x] There is no commented out code in this PR.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -25,7 +25,7 @@ On some networks discovery doesn't work reliably, in that case if you can find t
 .. code-block:: python
 
     >>> import pywemo
-    >>> url = pywemo.setup_url_for_address("192.168.1.192", None)
+    >>> url = pywemo.setup_url_for_address("192.168.1.192")
```
Comment (esev) [inline README.rst:31]: WDYT about changing the documentation after the next release? That way the examples match what folks are able to download from pypi.

**Comment by Spectre5** [comment]
Ya I considered that issue too.  Maybe we just sit on this PR until that time and leave it open as a reminder?

File: README.rst (hunk 1 above)
Comment (Spectre5) [reply README.rst:31]: Oops, see reply in main comments.

**Comment by esev** [comment]
> Maybe we just sit on this PR until that time and leave it open as a reminder?

That sounds good. Let's do the change to discovery.py in another PR so it can be merged now. Then keep this open for after the next release.

**Pull request merged**